### PR TITLE
Migrate vapor pressure batch 11

### PR DESCRIPTION
**Pull request by sumitnegi7**
Review batch10 before reviewing this batch commented in the file from where batch11 starts

### CODE REVIEW

File: src/data/vapor-pressure-constants.ts [hunk 1]
```diff
@@ -3629,6 +3629,1551 @@ export const DIPPRVaporPressureDictionary: Record<string, DIPPRVaporPressureProp
...
+    CAS: "110-66-7",
+    C1: 67.309,
+    C2: -6880.8,
+    C3: -6.4449,
+    C4: 1.01e-17,
+    C5: 6,
+    minimumTemperature: 197.45,
+    maximumTemperature: 598,
+    pressureAtMinimumTemperature: 0.201,
+    pressureAtMaximumTemperature: 3473000
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:4306]: Why change the format?
 3.473E+06

File: src/data/vapor-pressure-constants.ts [hunk 2]
```diff
@@ -3629,6 +3629,1551 @@ export const DIPPRVaporPressureDictionary: Record<string, DIPPRVaporPressureProp
...
+    maximumTemperature: 653,
+    pressureAtMaximumTemperature: 4.063e6
+  },
+
+  PhthalicAnhydride: {
+    name: "Phthalic anhydride",
+    formula: "C8H4O3",
+    CAS: "85-44-9",
+    C1: 126.5,
+    C2: 12551,
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:4389]: −12,551

File: src/data/vapor-pressure-constants.ts [hunk 3]
```diff
@@ -3629,6 +3629,1551 @@ export const DIPPRVaporPressureDictionary: Record<string, DIPPRVaporPressureProp
...
+    C3: 15.002,
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:4390]: −15.002

File: src/data/vapor-pressure-constants.ts [hunk 4]
```diff
@@ -3629,6 +3629,1551 @@ export const DIPPRVaporPressureDictionary: Record<string, DIPPRVaporPressureProp
...
+    formula: "C3H6O2",
+    CAS: "79-09-4",
+    C1: 54.552,
+    C2: -7149.4,
+    C3: -4.2769,
+    C4: 1.18e-18,
+    C5: 6,
+    minimumTemperature: 252.45,
+    maximumTemperature: 600.81,
+    pressureAtMinimumTemperature: 13.1e1,
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:4500]: 1.31E+01

File: src/data/vapor-pressure-constants.ts [hunk 5]
```diff
@@ -3629,6 +3629,1551 @@ export const DIPPRVaporPressureDictionary: Record<string, DIPPRVaporPressureProp
...
+    C3: -5.8595,
+    C4: 2.06e-17,
+    C5: 6,
+    minimumTemperature: 159.95,
+    maximumTemperature: 536.6,
+    pressureAtMinimumTemperature: 6.51e-2,
+    pressureAtMaximumTemperature: 4.627e6
+  },
+
+  "12PropyleneGlycol": {
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:4626]: "1,2PropyleneGlycol": {

File: src/data/vapor-pressure-constants.ts [hunk 6]
```diff
@@ -3629,6 +3629,1551 @@ export const DIPPRVaporPressureDictionary: Record<string, DIPPRVaporPressureProp
...
+  },
+
+  oXylene: {
+    name: "o-Xylene",
+    formula: "C8H10",
+    CAS: "95-47-6",
+    C1: 90.405,
+    C2: -7955.2,
+    C3: -10.086,
+    C4: 0.0000059594,
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:5156]: Why change the format?
5.9594E-06

File: src/data/vapor-pressure-constants.ts [hunk 7]
```diff
@@ -3629,6 +3629,1551 @@ export const DIPPRVaporPressureDictionary: Record<string, DIPPRVaporPressureProp
...
+    CAS: "75-94-5",
+    C1: 54.571,
+    C2: -5561.5,
+    C3: -4.712,
+    C4: 1.07e-17,
+    C5: 6,
+    minimumTemperature: 178.35,
+    maximumTemperature: 543.15,
+    pressureAtMinimumTemperature: 3.54e-1,
+    pressureAtMaximumTemperature: 3.508e6
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:5116]: 3.058E+06

File: src/data/vapor-pressure-constants.ts [hunk 8]
```diff
@@ -3629,6 +3629,1551 @@ export const DIPPRVaporPressureDictionary: Record<string, DIPPRVaporPressureProp
...
+    C3: -69.22,
+    C4: 0.000027381,
+    C5: 2,
+    minimumTemperature: 398.4,
+    maximumTemperature: 846,
+    pressureAtMinimumTemperature: 8.5,
+    pressureAtMaximumTemperature: 3.41e6
+  },
+
+  "246Trinitrotoluene": {
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:5014]: "2,4,6Trinitrotoluene": {

File: src/data/vapor-pressure-constants.ts [hunk 9]
```diff
@@ -3629,6 +3629,1551 @@ export const DIPPRVaporPressureDictionary: Record<string, DIPPRVaporPressureProp
...
+    C3: -9.1858,
+    C4: 6.4703e-6,
+    C5: 2,
+    minimumTemperature: 172.22,
+    maximumTemperature: 573.5,
+    pressureAtMinimumTemperature: 1.68e-2,
+    pressureAtMaximumTemperature: 2.812e6
+  },
+
+  "135Trinitrobenzene": {
```
Comment (saikat-samanta) [inline src/data/vapor-pressure-constants.ts:4999]: "1,3,5Trinitrobenzene": {
so on...

File: src/data/vapor-pressure-constants.ts (hunk 3 above)
Comment (sumitnegi7) [reply src/data/vapor-pressure-constants.ts:4390]: Before line 4579 changes are made in batch10

**Review by saikat-samanta — APPROVED** [review]
All good👍

**Review by qur786 — APPROVED** [review]
all good

File: src/data/vapor-pressure-constants.ts [hunk 10]
```diff
@@ -3629,6 +3629,1551 @@ export const DIPPRVaporPressureDictionary: Record<string, DIPPRVaporPressureProp
...
+    C3: -3.6469,
+    C4: 1.8e-17,
+    C5: 6,
+    minimumTemperature: 197.67,
+    maximumTemperature: 430.75,
+    pressureAtMinimumTemperature: 1.67e3,
+    pressureAtMaximumTemperature: 7.86e6
+  },
+
+  SulfurHexafluoride: {
```
Comment (qur786) [inline src/data/vapor-pressure-constants.ts:4716]: @maneetgoyal @sumitnegi7  we need to change interface accordingly for SulfurHexafluoride , some of its data is missing

**Review by qur786 — CHANGES_REQUESTED** [review]

**Review by qur786 — APPROVED** [review]
all good ,A suggestion , should add a test case for SulfurHexafluoride as well

**Pull request merged**